addon/mixins/child.js

```diff
@@ -55,11 +55,18 @@ export default Mixin.create({
     }
   },
 
+  notifyParentOfInsert: on('didInsertElement', function() {
```

Comment:
I went with `on('didInsertElement'…` explicitly to avoid subclasses being able to override this and break the chain. I'll swap it.